test/checks/pdbMinAvailableGreaterThanHPAMinReplicas/failure-gt-percent.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: zookeeper
+spec:
+  replicas: 10
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: zookeeper
+        foo: bar
+    spec:
+      containers:
+        - name: zookeeper
+          image: zookeeper
+---
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: zookeeper-pdb
+spec:
+  minAvailable: 60% # 0.6 * 10 = 6
```

Comment:
If omitting the replicas in the deployment, this will be driven by the minReplicas of the HPA - so I think either change minReplicas in the HPA to 10 so the math is simpler

Reply:
I'm calculating the percentage value based on `deployment.Replicas` [here](https://github.com/FairwindsOps/polaris/pull/1057/files#diff-79fdf9ca8b64facbd6edb0d64e79299cd17e59bb2733d02760c0fb812a3d869aR63) - should I fall back to `hpa.MinValue` in that case?

- looks like `deployment.Replicas` is never nullable when fetching the resource from k8s
- maybe it can only be missing when analyzing the file